packages/puppeteer-core/src/bidi/HTTPRequest.ts

```diff
@@ -166,14 +168,14 @@ export class BidiHTTPRequest extends HTTPRequest {
     throw new UnsupportedOperation();
   }
 
-  override abort(): never {
-    throw new UnsupportedOperation();
+  override async abort(): Promise<void> {
+    return await this.#request.failRequest();
   }
 
-  override respond(
+  override async respond(
     _response: Partial<ResponseForRequest>,
     _priority?: number
-  ): never {
+  ): Promise<void> {
     throw new UnsupportedOperation();
```

Comment:
also a thought: should we delay making a changelog update until most of the interception features are supported?

Reply:
Let's probably do that (mark this a chore, and mark the last change as a feat + update webdriver support page), since it will read confusing in the changelog.